File: core/src/main/java/com/cc/core/command/Command.java
package com.cc.core.command;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Command {
    @SerializedName("id")
    private String id;
    @SerializedName("key")
    private String key;
    @SerializedName("args")
    private List<? extends Object> args;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public List<? extends Object> getArgs() {
        return args;
    }

    public void setArgs(List<? extends Object> args) {
        this.args = args;
    }
}

File: core/src/main/java/com/cc/core/command/Command.kt
package com.cc.core.command

import com.google.gson.annotations.SerializedName

class Command {
  @SerializedName("id")
  private var id: String = ""
  @SerializedName("key")
  private var key: String = ""
  @SerializedName("args")
  private var args: List<Any>? = null

  fun getId(): String {
    return id
  }

  fun setId(id: String) {
    this.id = id
  }

  fun getKey(): String {
    return key
  }

  fun setKey(key: String) {
    this.key = key
  }

  fun getArgs(): List<Any>? {
    return args
  }

  fun setArgs(args: List<Any>) {
    this.args = args
  }
}